engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/compile/CompiledProcess.scala

```diff
@@ -24,9 +25,12 @@ object CompiledProcess {
              ): ValidatedNel[ProcessCompilationError, CompiledProcess] = {
     val servicesDefs = definitions.services
 
-    val expressionCompiler = ExpressionCompiler.withOptimization(userCodeClassLoader, definitions.expressionConfig)
+    val dictRegistryFactory = DictServicesFactoryLoader.justOne(userCodeClassLoader)
```

Comment:
ugh... I *don't* think this should be here... We shouldn't mess with classloaders here...

Reply:
Why? Which level will be better? Near creating of `ProcessConfigCreator`?

Reply:
ok, for now let's just exctract creating registry to a method, to make it more visible, ok?

Reply:
extracted